diff --git a/node_modules/@ffmpeg/ffmpeg/dist/esm/classes.js b/node_modules/@ffmpeg/ffmpeg/dist/esm/classes.js
index 2886dbd..150b189 100644
--- a/node_modules/@ffmpeg/ffmpeg/dist/esm/classes.js
+++ b/node_modules/@ffmpeg/ffmpeg/dist/esm/classes.js
@@ -100,9 +100,18 @@ export class FFmpeg {
      */
     load = (config = {}, { signal } = {}) => {
         if (!this.#worker) {
-            this.#worker = new Worker(new URL("./worker.js", import.meta.url), {
-                type: "module",
-            });
+            // this.#worker = new Worker(new URL("./worker.js", import.meta.url), {
+            //     type: "module",
+            // });
+            if( config.workerURL ){
+                this.#worker = new Worker( config.workerURL, {
+                  type: "module"
+                });
+              } else {
+                this.#worker = new Worker(new URL("./worker.js", import.meta.url), {
+                  type: "module",
+                });
+              }
             this.#registerHandlers();
         }
         return this.#send({
